vars/inside.groovy

```diff
@@ -13,6 +13,11 @@ def call(Map parameters = [:], body) {
 
     slave(parameters) {
         node(label) {
+            sh '''
```

Comment:
I don't like the fact that handling of maven repositories / settings is leaking to non-maven related functions like this.